cmd/initializer/main.go

```diff
@@ -13,7 +13,9 @@ import (
 var rootCmd = &cobra.Command{
 	Short: "initializer",
 	Long:  `initializer takes a directory of OLM manifests and outputs a sqlite database containing them`,
-
+	PersistentPreRun: func(_ *cobra.Command, _ []string) {
+		sqlite.LogSqliteDeprecation()
+	},
```

Comment:
Should we also surface this information when interacting with `--help` type commands?

```bash
$ ./bin/initializer 
WARN[0000] DEPRECATION NOTICE:
Sqlite-based catalogs and their related subcommands are deprecated. Support for
them will be removed in a future release. Please migrate your catalog workflows
to the new file-based catalog format.
 
INFO[0000] loading Bundles                               dir=manifests
INFO[0000] directory                                     dir=manifests file=manifests load=bundles
INFO[0000] directory                                     dir=manifests file=etcd load=bundles
INFO[0000] directory                                     dir=manifests file=0.6.1 load=bundles
```

Seems to display that fine, but `./bin/initializer --help` doesn't have any mentions of the sqlite-based index images in the output:

```bash
$  ./bin/initializer  --help
initializer takes a directory of OLM manifests and outputs a sqlite database containing them

Usage:
   [flags]

Flags:
  -h, --help               help for this command
  -m, --manifests string   relative path to directory of manifests (default "manifests")
  -o, --output string      relative path to a sqlite file to create or overwrite (default "bundles.db")
      --permissive         allow registry load errors
```